moveit_ros/moveit_servo/config/test_config_panda.yaml

```diff
@@ -2,20 +2,20 @@
 # Modify all parameters related to servoing here
 ###############################################
 
-## Properties of incoming commands
-command_in_type: "speed_units" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
-scale:
-  # Scale parameters are only used if command_in_type=="unitless"
-  linear:  0.6  # Max linear velocity. Unit is [m/s]. Only used for Cartesian commands.
-  rotational:  0.3 # Max angular velocity. Unit is [rad/s]. Only used for Cartesian commands.
-  # Max joint angular/linear velocity. Only used for joint commands on joint_command_in_topic.
-  joint: 0.01
-
 # Optionally override Servo's internal velocity scaling when near singularity or collision (0.0 = use internal velocity scaling)
 # override_velocity_scaling_factor = 0.0 # valid range [0.0:1.0]
 
 ## Properties of outgoing commands
-publish_period: 0.034  # 1/Nominal publish rate [seconds]
+publish_period: 0.02  # 1/Nominal publish rate [seconds]
+
+incoming_command_timeout: 0.5 # seconds
```

Comment:
Can this be brought back to its original 0.1 seconds for the timeout, or is there valid reason to increase it?